docroot/themes/custom/uids_base/scss/admin.scss

```diff
@@ -890,15 +890,18 @@ div[data-drupal-selector="edit-moderation-state-0"] {
 
   .paragraphs-dropdown-toggle {
     width: 26px !important;
+    height: 26px !important;
+    border: none !important;
     background: url(/modules/contrib/paragraphs/icons/icon-actions.svg) no-repeat center !important;
   }
 
   .paragraphs-dropdown-actions {
     border: none;
-    top: 0;
+    top: 1px;
```

Comment:
@bspeare I made this small adjustment, because there was a pixel difference between the two buttons (dropdown actions overlay non-dropdown actions), and it led to a weird overlap. I feel like there should be a better way to handle this that I may be overlooking, though.